### PR TITLE
fix #1606

### LINKED ISSUE #1606: Button "Create evaluation" visible for reviewers
As a reviewer viewing the overview of a semester, the list shows buttons to "create evaluation" and "create single result".
When clicking the button, the system presents me with an error view because I don't have the appropriate permissions.
I expect the system to only show me buttons for the actions that I have permissions for.

### PR DESCRIPTION
**Pull request by fidoriel**
won't show buttons to users without permissions

### CODE REVIEW

**Review by He3lixxx — APPROVED** [review]
Cool, thanks!

:heavy_check_mark: code looks good to me

**Review by He3lixxx — APPROVED** [review]
Still good :D

File: evap/staff/templates/staff_semester_view.html [hunk 1]
```diff
@@ -154,12 +154,14 @@ <h3>
             <div class="tab-pane show active" id="evaluations" role="tabpanel">
                 <div class="row align-items-center mb-3">
                     <div class="col-9">
+                        {% if request.user.is_manager %}
                         <a class="btn btn-sm btn-dark" href="{% url 'staff:evaluation_create' semester.id %}">
                             {% trans 'Create evaluation' %}
                         </a>
                         <a class="btn btn-sm btn-dark ms-2" href="{% url 'staff:single_result_create' semester.id %}">
                             {% trans 'Create single result' %}
                         </a>
```
Comment (janno42) [inline evap/staff/templates/staff_semester_view.html:163]: please increase the indentation here and for the other two changes below

**Review by janno42 — CHANGES_REQUESTED** [review]

**Review by janno42 — APPROVED** [review]
thank you for your first evap pr 🎉 

✔️ Meets requirements
✔️ UI functionality checked

**Pull request merged**